mindjunkies/lecture/views.py

```
@@ -18,7 +21,11 @@
 from mindjunkies.courses.models import Course, Module
 
 from .forms import LectureForm, LecturePDFForm, LectureVideoForm, ModuleForm
-from .models import Lecture, LecturePDF, LectureVideo
+from .models import Lecture, LecturePDF, LectureVideo, LectureCompletion, LastVisitedModule
+from django.utils.timezone import localtime, now
```

Ruff (0.8.2)

Redefinition of unused `localtime` from line 15

Remove definition: `localtime`

(F811)